src/promptflow-devkit/promptflow/_sdk/_configuration.py

```diff
@@ -52,12 +52,13 @@ class Configuration(object):
     TRACE_DESTINATION = "trace.destination"
     _instance = None
 
-    def __init__(self, overrides=None):
+    def __init__(self, overrides=None, skip_validation=False):
```

Comment:
where do we use this param? did we set it to True some where?

Reply:
We have removed this parameter.